docs/userGuide/plugins/dataTable.md

```diff
@@ -0,0 +1,66 @@
+### Plugin: DataTable
+
+The [DataTable](https://datatables.net) plugin enhances the functionality of tables in your MarkBind site by integrating the DataTables library. It allows you to add searching and sorting capabilities to your tables with minimal configuration. The necessary CSS and JavaScript files are already included in the project, so no additional CDN or plugin context configuration is required.
+
+To enable this plugin, simply add `dataTable` to your site's plugins:
+
+```js {heading="site.json"}
+{
+  ...
+  "plugins": [
+    "dataTable"
+  ]
+}
+```
+
+To create a table with DataTable features, use one of the following syntaxes:
+
+{{ icon_example }} Sortable Table:
```

Comment:
Small nit! Could you put the "Sortable Table:" title below the example icon, similarly for the examples below as well. It looks a little out of place beside the icon imo.

Reply:
I think this is consistent with the formatting for the rest of the page 
![image](https://github.com/MarkBind/markbind/assets/77204346/0c0d6460-c962-4b7e-ba5b-b1eadf20a1ec)

Reply:
ohh then it's alright thanks!